ethcore/types/src/transaction/transaction.rs

```diff
@@ -235,7 +239,7 @@ impl Transaction {
 	}
 
 	/// Legacy EIP-86 compatible empty signature.
-	/// Used only for tests.
+	#[cfg(any(test, feature = "test-helpers"))]
 	pub fn null_sign(self, chain_id: u64) -> SignedTransaction {
```

Comment:
So this is allowed in tests but not in production? I think this merits a comment.

Reply:
They were not allowed in production, we had `verify_basic` check that returned an error because eip86transition is not used anymore. I added `Legacy` for that, what do you think needs more clarification?

Reply:
> what do you think needs more clarification?

We do not support `EIP-86` anymore (if we ever did), and now we're removing the last pieces of what support there used to be, but we still have this method available for tests to use. I'm not clear on why we need tests to be able to call this when no other code can. Is it to support running legacy tests? If so, it's worth explaining imo. Is it for json tests only? If so, also worth mentioning.